docs/product-user-guide/src/main/asciidoc/project-deploying-con.adoc

```diff
@@ -0,0 +1,14 @@
+[id='project-deploying-con_{context}']
+= Deploying a project
+
+The deployment process can vary based on the needs of your infrastructure.
+
+In a simple deployment of {PRODUCT}, there is one {CENTRAL} and one {KIE_SERVER}. You can use {CENTRAL} to develop your business assets and also to manage the {KIE_SERVER}. You can build your project in {CENTRAL} and it is automatically deployed onto the {KIE_SERVER}. To enable automatic deployment, {CENTRAL} includes a built-in Maven repository. You can use {CENTRAL} to manage the {KIE_SERVER}, deploying, removing, starting, and stopping any of the services and their project versions that you have built.
```

Comment:
This is a little clunky:
You can build your project in {CENTRAL} and it is automatically deployed onto the {KIE_SERVER}.

Reply:
Yeah, I struggled with this section in general. I tweaked some things. Thanks.